addons/mail/static/src/new/chat/chat_window_container.js

```diff
@@ -11,9 +11,27 @@ export class ChatWindowContainer extends Component {
 
     setup() {
         this.messaging = useMessaging();
+        this.state = useState({
+            isActive: false,
+        });
+        const updateIsActiveState = () => {
+            this.state.isActive = !this.messaging.state.discuss.isActive;
+        };
+        this.env.bus.addEventListener("ACTION_MANAGER:UI-UPDATED", updateIsActiveState);
+        onWillDestroy(() => {
+            this.env.bus.removeEventListener("ACTION_MANAGER:UI-UPDATED", updateIsActiveState);
+        });
```

Comment:
I see some `useBus(this.env.bus, "ACTION_MANAGER:UI-UPDATED")`. Could be DRYer to do that.